pkg/api/v1/routes/handlers_test.go

```diff
@@ -393,6 +331,17 @@ func TestAddServer(t *testing.T) {
 			server.ServeHTTP(recorder, tc.request(t))
 
 			tc.assertResponse(t, recorder)
+
+			// Assert all expectations
```

Comment:
Why are we doing this here and not in other tests? The GoMock ‘finisher’ asserts the mock expectations.